make install-tools fails on third party library

Local and on Travis CI during make install-tools the following error gets emitted:
```
package github.com/kisielk/errcheck/...
	imports github.com/kisielk/errcheck
	imports github.com/kisielk/errcheck/internal/errcheck
	imports go/types: unrecognized import path "go/types"

make: *** [install-tools] Error 1
```
Prevents #106

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,6 @@
 language: go
+go:
+  - 1.5
 
 addons:
   apt:

--- a/function.go
+++ b/function.go
@@ -88,7 +88,7 @@ func handleFunctionCursor(cursor clang.Cursor) *Function {
 
 	numParam := int(cursor.NumArguments())
 	for i := 0; i < numParam; i++ {
-		param := cursor.Argument(uint16(i))
+		param := cursor.Argument(uint32(i))
 
 		p := FunctionParameter{
 			CName: param.DisplayName(),

--- a/scripts/vagrant-environment.sh
+++ b/scripts/vagrant-environment.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export GO_VERSION=1.4.3
+export GO_VERSION=1.5.3
 
 # Install Go
 mkdir -p $HOME/go

